conda_build/cli/main_render.py

```diff
@@ -101,6 +101,14 @@ def get_render_parser():
         on clobbered fields.""",
         dest='clobber_sections_file',
     )
+    p.add_argument(
+        '-m', '--variant-config-files',
+        dest='variant_config_files',
```

Comment:
This is redundant.  Argparse automatically replaces - with _, so the dest is already variant_config_files.

Is m an unused one-letter flag?  If so, cool.  AFAICT, it is used in the inspect command, but that should not conflict here.  --variant-config-files is rather long-winded.